### PR TITLE
Aruba 2930M webauth/dacl support

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Aruba 2930M webauth and dacl support

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## New Features
* Aruba 2930M Web Authentication and Dynamic ACL support

### CODE REVIEW

File: lib/pf/Switch/Aruba/2930M.pm [hunk 1]
```diff
@@ -69,31 +88,203 @@ sub returnRadiusAccessAccept {
...
+            secret => $self->{'_radiusSecret'},
+            LocalAddr => $self->deauth_source_ip($send_disconnect_to),
+        };
+
+        $logger->debug("[$self->{'_ip'}] Network device supports roles. Evaluating role to be returned.");
+        my $roleResolver = pf::roles::custom->instance();
+        my $role = $roleResolver->getRoleForNode($mac, $self);
 
-Copyright (C) 2005-2021 Inverse inc.
+        # transforming MAC to the expected format 00-11-22-33-CA-FE
```
Comment (JeGoi) [inline lib/pf/Switch/Aruba/2930M.pm]: If lc($mac), then 00-11-22-33-ca-fe, correct?

**Review by JeGoi — COMMENTED** [review]
Just one question about the typo of a mac address.
Then it is fine for me

**Review by JeGoi — APPROVED** [review]
Ok sounds perfect to me.
thanks

**Comment by extrafu** [comment]
@JeGoi You *must* update the NEWS.asciidoc file post-merge with what is specified in the "NEWS file entries" from this PR.

**Comment by JeGoi** [comment]
Thanks @extrafu It is done.

**Pull request merged**